packages/core/src/index.js.flow

```diff
@@ -74,7 +74,7 @@ declare export function continueWith <A> (f: (any) => A): (s: Stream<A>) => Stre
 
 declare export function switchLatest <A> (s: Stream<Stream<A>>): Stream<A>
 
-declare export function merge <A> (...ss: Array<Stream<A>>): Stream<A>
+declare export function merge <A> (sa: Stream<A>, sb: Stream<A>): Stream<A>
```

Comment:
Types look good :+1:.  In a few places, we've already used `sb` as a name for `Stream<B>`.  How about we call these two streams `s1` and `s2`, or `sa1` and `sa2`, just to avoid any potential confusion.

Also need to add the other curried type def variant here to allow partial application:

```js
declare export function merge <A> (sa: Stream<A>): (sb: Stream<A>) => Stream<A>
```

For two parameters, it's not too bad ... but for 3 it starts to get tedious 😞 See below.